src/workloads/query/LookupColocatedData.yml

```diff
@@ -0,0 +1,368 @@
+      OnlyActiveInPhase: 12
+      MetricsName: ShardedLocalFieldForeignField1k
+      Coll: *ShardedLocal
+      BatchSize: *BatchSize1k
+
+- ActorFromTemplate:
+    TemplateName: SelfLookupLocalFieldForeignField
+    TemplateParameters:
+      OnlyActiveInPhase: 13
+      MetricsName: ShardedLocalFieldForeignField5k
+      Coll: *ShardedLocal
+      BatchSize: *BatchSize5k
+
+- ActorFromTemplate:
+    TemplateName: SelfLookupLocalFieldForeignField
+    TemplateParameters:
+      OnlyActiveInPhase: 14
+      MetricsName: ShardedLocalFieldForeignField15k
+      Coll: *ShardedLocal
+      BatchSize: *BatchSize15k
+
+- ActorFromTemplate:
+    TemplateName: SelfLookupLetPipeline
+    TemplateParameters:
+      OnlyActiveInPhase: 15
+      MetricsName: ShardedLetPipeline1k
+      Coll: *ShardedLocal
+      BatchSize: *BatchSize1k
+
+- ActorFromTemplate:
+    TemplateName: SelfLookupLetPipeline
+    TemplateParameters:
+      OnlyActiveInPhase: 16
+      MetricsName: ShardedLetPipeline5k
+      Coll: *ShardedLocal
+      BatchSize: *BatchSize5k
+
+- ActorFromTemplate:
+    TemplateName: SelfLookupLetPipeline
+    TemplateParameters:
+      OnlyActiveInPhase: 17
+      MetricsName: ShardedLetPipeline15k
+      Coll: *ShardedLocal
+      BatchSize: *BatchSize15k
+
+#
+# Sharded, all on same shard
+#
+- ActorFromTemplate:
+    TemplateName: SelfLookupLocalFieldForeignField
+    TemplateParameters:
+      OnlyActiveInPhase: 18
+      MetricsName: ShardedAllOnOneShardLocalFieldForeignField1k
+      Coll: *ShardedLocalAllOnOneShard
+      BatchSize: *BatchSize1k
+
+- ActorFromTemplate:
+    TemplateName: SelfLookupLocalFieldForeignField
+    TemplateParameters:
+      OnlyActiveInPhase: 19
+      MetricsName: ShardedAllOnOneShardLocalFieldForeignField5k
+      Coll: *ShardedLocalAllOnOneShard
+      BatchSize: *BatchSize5k
+
+- ActorFromTemplate:
+    TemplateName: SelfLookupLocalFieldForeignField
+    TemplateParameters:
+      OnlyActiveInPhase: 20
+      MetricsName: ShardedAllOnOneShardLocalFieldForeignField15k
+      Coll: *ShardedLocalAllOnOneShard
+      BatchSize: *BatchSize15k
+
+- ActorFromTemplate:
+    TemplateName: SelfLookupLetPipeline
+    TemplateParameters:
+      OnlyActiveInPhase: 21
+      MetricsName: ShardedAllOnOneShardLetPipeline1k
+      Coll: *ShardedLocalAllOnOneShard
+      BatchSize: *BatchSize1k
+
+- ActorFromTemplate:
+    TemplateName: SelfLookupLetPipeline
+    TemplateParameters:
+      OnlyActiveInPhase: 22
+      MetricsName: ShardedAllOnOneShardLetPipeline5k
+      Coll: *ShardedLocalAllOnOneShard
+      BatchSize: *BatchSize5k
+
+- ActorFromTemplate:
+    TemplateName: SelfLookupLetPipeline
+    TemplateParameters:
+      OnlyActiveInPhase: 23
+      MetricsName: ShardedAllOnOneShardLetPipeline15k
+      Coll: *ShardedLocalAllOnOneShard
+      BatchSize: *BatchSize15k
+
+AutoRun:
+- When:
+    branch_name:
+      $neq:
+      - v4.0
+      - v4.2
+      - v4.4
+      - v5.0
+    mongodb_setup:
+      $eq:
```

Comment:
Presumably this is more like a $in?

Reply:
Yes. This is just how it is done everywhere, see https://github.com/mongodb/genny/blob/master/docs/using.md#configuring-autorun